source/MRMesh/MRCNCMachineSettings.cpp

```diff
@@ -3,26 +3,18 @@
 namespace MR
 {
 
-void CNCMachineSettings::setRotationParams( RotationAxisName paramName, const Vector3f& rotationAxis )
+void CNCMachineSettings::setRotationAxis( RotationAxisName paramName, const Vector3f& rotationAxis )
 {
     const int intParamName = int( paramName );
-    const bool validParamName = intParamName < int( RotationAxisName::Count );
-    if ( !validParamName )
-        return;
-
     if ( rotationAxis.lengthSq() < 0.01f )
         return;
```

Comment:
is this way to invalidate `rotationAxes_[paramName]`? then why return

Reply:
no, if we try set too short vector, we ignore it. (keep old value)